lib/ansible/modules/network/avi/avi_pool.py

```diff
@@ -30,61 +30,74 @@
             - The state that should be applied on the entity.
         default: present
         choices: ["absent", "present"]
+        type: str
     avi_api_update_method:
         description:
             - Default method for object update is HTTP PUT.
             - Setting to patch will override that behavior to use HTTP PATCH.
         version_added: "2.5"
         default: put
         choices: ["put", "patch"]
+        type: str
     avi_api_patch_op:
         description:
             - Patch operation to use when using avi_api_update_method as patch.
         version_added: "2.5"
         choices: ["add", "replace", "delete"]
+        type: str
     a_pool:
         description:
             - Name of container cloud application that constitutes a pool in a a-b pool configuration, if different from vs app.
             - Field deprecated in 18.1.2.
+        type: str
     ab_pool:
         description:
             - A/b pool configuration.
             - Field deprecated in 18.1.2.
+        type: dict
     ab_priority:
         description:
             - Priority of this pool in a a-b pool pair.
             - Internally used.
             - Field deprecated in 18.1.2.
+        type: int
     analytics_policy:
         description:
             - Determines analytics settings for the pool.
             - Field introduced in 18.1.5, 18.2.1.
         version_added: "2.9"
+        type: dict
     analytics_profile_ref:
         description:
             - Specifies settings related to analytics.
             - It is a reference to an object of type analyticsprofile.
             - Field introduced in 18.1.4,18.2.1.
         version_added: "2.9"
+        type: str
     apic_epg_name:
         description:
             - Synchronize cisco apic epg members with pool servers.
+        type: str
     application_persistence_profile_ref:
         description:
             - Persistence will ensure the same user sticks to the same server for a desired duration of time.
             - It is a reference to an object of type applicationpersistenceprofile.
+        type: str
     autoscale_launch_config_ref:
         description:
             - If configured then avi will trigger orchestration of pool server creation and deletion.
-            - It is only supported for container clouds like mesos, openshift, kubernetes, docker, etc.
+            - It is only supported for container clouds like mesos, opensift, kubernates, docker etc.
```

Comment:
```suggestion
            - It is only supported for container clouds like mesos, openshift, kubernates, docker etc.
```
The `h` was supposed to be there.